### PR TITLE
Bug/171 env var next core

### LINKED ISSUE #171: Env var for next core use value for main core
```javascript
if (process.env.PERSEO_NEXT_URL) {
        config.nextCore.rulesURL = process.env.PERSEO_CORE_URL + '/perseo-core/rules';
        config.nextCore.noticesURL = process.env.PERSEO_CORE_URL + '/perseo-core/events';
}
```

### PR DESCRIPTION
**Pull request by crbrox**
closes #171, #172 

@fgalan 

FYI:   @AlvaroVega

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -2,3 +2,5 @@
 - Add: Allow notices and rules paths to be configured by environment variables (issue #161)
 - Fix: align and fix log traces, add comp field (issue #166)
 - Add: add 'from' field in log traces (issue #166)
+- Fix: env var for next core bug (issue #171)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Maybe the new env var should be described in the documentation along with all the other env vars?

File: config.js [hunk 2]
```diff
@@ -91,8 +91,10 @@ config.perseoCore = {
  * NEXT EPL core options (with HA)
  */
 config.nextCore = {
+    /*
```
Comment (fgalan) [inline config.js]: I would add a comment in the following style:

"Note this parameters is empty, so Perseo will not use HA by default"

Comment (crbrox) [inline config.js]: Fixed a457671

**Comment by crbrox** [comment]
(Rebase to avoid conflict with develop)

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (crbrox) [inline CHANGES_NEXT_RELEASE]: Already described in https://github.com/telefonicaid/perseo-fe/blob/develop/documentation/configuration.md#environment-variables-configuration

NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**